Marlin/Marlin_main.cpp

```diff
@@ -8276,8 +8271,8 @@ void ok_to_send() {
       SERIAL_ECHOPAIR(" L=", L);
       SERIAL_ECHOPAIR(" R=", R);
       SERIAL_ECHOLNPAIR(" offset=", offset);
+      last_offset = offset;
```

Comment:
This line should be left outside of the braces where it is. The intent is to compare the previous offset to the current one (not just the last offset that was too large).